### PR TITLE
Ofmcc 1110 update organization information

### PR DESCRIPTION
**Pull request by jgstorey**
Code QA notes:

1. Future fields that will be updatable are implemented but commented out.
2. To achieve the new UI i used to v-cards stacked side by side.
3. Tested for phone resolution... all fields wrap but the 3 new editable fields extend into the card. Tried to resolve but began spinning wheels so stopped.

### CODE REVIEW

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: backend/src/components/organizations.js [hunk 1]
```diff
@@ -26,6 +26,17 @@ async function getOrganizationFacilities(req, res) {
   }
 }
 
+async function updateOrganization(req, res) {
+  let organization = new MappableObjectForBack(req.body, OrganizationMappings).toJSON()
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply backend/src/components/organizations.js:30]: Fixed.

File: backend/src/components/organizations.js [hunk 2]
```diff
@@ -26,6 +26,17 @@ async function getOrganizationFacilities(req, res) {
...
+  try {
+    let orgResponse = await patchOperationWithObjectId('accounts', req.params.organizationId, organization)
+    orgResponse = new MappableObjectForFront(orgResponse, OrganizationMappings)
+    return res.status(HttpStatus.OK).json(orgResponse)
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply backend/src/components/organizations.js:34]: Fixed.

File: frontend/src/components/organizations/OrganizationInfo.vue [hunk 3]
```diff
@@ -2,190 +2,227 @@
...
-                    {{ organizationInfo?.streetAddress1 }}
+        <v-form ref="organizationForm" v-model="isFormComplete">
+          <v-row no-gutters>
+            <v-col cols="12" md="6" lg="6">
+              <v-card variant="flat">
+                <v-row no-gutters>
+                  <v-col cols="12" sm="3" md="3" lg="3" class="ma-2">
+                    <AppLabel>Organization legal name:</AppLabel>
                   </v-col>
+                  <v-col cols="12" sm="8" md="8" lg="8" class="ma-2">
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/organizations/OrganizationInfo.vue:13]: Confirmed, it fixed it.

File: frontend/src/components/organizations/OrganizationInfo.vue [hunk 4]
```diff
@@ -2,190 +2,227 @@
...
                 </v-row>
-              </v-col>
-            </v-row>
-          </v-col>
-        </v-row>
+              </v-card>
+            </v-col>
+          </v-row>
+          <v-row v-if="editMode">
+            <v-col cols="12" class="d-flex justify-center">
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/organizations/OrganizationInfo.vue:220]: Fixed.

File: frontend/src/services/organizationService.js [hunk 5]
```diff
@@ -23,4 +23,15 @@ export default {
...
     }
   },
+
+  async updateOrganization(organizationId, organization) {
+    try {
+      if (!organizationId) return
+      const response = await ApiService.apiAxios.put(ApiRoutes.ORGANIZATIONS + '/' + organizationId, organization)
+      return response?.data
+    } catch (error) {
+      console.log(`Failed to update organization by organization/account id - ${error}`)
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/services/organizationService.js:33]: Given its for the log file as well as bridges the gap between our backend and CRM i thought i was useful info. organizationid (our backend) / accountid (CRM). Viet initially created this service and i used for consistency as i got the meaning. Let me know if you still want updated.

File: frontend/src/components/organizations/OrganizationInfo.vue [hunk 6]
```diff
@@ -2,190 +2,227 @@
...
-                  <v-col cols="12" md="6" lg="5" xl="4">
-                    <AppLabel>City:</AppLabel>
+                <v-row no-gutters>
+                  <v-col cols="12" sm="3" md="3" lg="3" class="ma-2">
+                    <AppLabel>Email address:</AppLabel>
                   </v-col>
-                  <v-col cols="12" md="6" lg="7" xl="8">
-                    {{ organizationInfo?.city }}
+                  <v-col cols="12" sm="8" md="8" lg="8" class="ma-2">
+                    <template v-if="editMode">
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/organizations/OrganizationInfo.vue]: Fixed.

File: frontend/src/services/organizationService.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/services/organizationService.js:33]: Fine with me.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**